### PR TITLE
Quarkus Native with Podman for Windows

### PR DESCRIPTION
**Pull request by Karm**
This is a blogpost about Quarkus Native with Podman for Windows.

Please, do not merge as it depends on some upstream PRs, namely:
https://github.com/quarkusio/quarkus/pull/33720
https://github.com/quarkusio/quarkus-quickstarts/pull/1283

The text contains "TODO" wrt to Q version and I will amend it later.
It is fit for review otherwise. 

It also fixes a dead link on: https://quarkus.io/blog/quarkus-native-neo4j-call-tree/ post.

### CODE REVIEW

**Comment by Karm** [comment]
https://quarkus-io-pr-1712-preview.surge.sh/blog/podman-for-windows/

**Comment by galderz** [comment]
Looks good, but this is WIP so not yet approving it?

**Comment by Karm** [comment]
> Looks good, but this is WIP so not yet approving it?


Ack. I am not planning on changing the text unless there are any new suggestions or objections.

I'll just wait till my fix is released in some upcoming Q 3.1.x version, remove "TODO" notes, test it again and it's done 👍

**Comment by Karm** [comment]
@maxandersen  @galderz I'd like this merged and published. I tested the flow with Quarkus 3.1.2.Final and it's sound. The needed changes are already in Quarkus.

**Comment by Karm** [comment]
Preview over here:

https://quarkus-website-pr-1712-preview.surge.sh/blog/podman-for-windows/

**Review by cescoffier — APPROVED** [review]

**Comment by cescoffier** [comment]
Hey @Karm !

Super nice blog post. 

Can you fix the _authors conflicts and update the publication date?

File: _posts/2023-06-23-podman-for-windows.adoc [hunk 1]
```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title: Podman for Windows
+date: 2023-06-23
```
Comment (gastaldi) [inline _posts/2023-06-23-podman-for-windows.adoc:4]: Bump the date to today before we merge this (also, please rename the file to match the date)

**Review by gastaldi — APPROVED** [review]
LGTM, but it needs a rebase before we merge it

File: _posts/2021-11-30-quarkus-native-neo4j-call-tree.adoc [hunk 2]
```diff
@@ -86,7 +86,7 @@ include::../assets/examples/posts/quarkus-native-neo4j-call-tree/import.cypher[]
 ----
 
 You can download the cypher script from
-link:../assets/examples/posts/quarkus-native-neo4j-call-tree/import.cypher[this link]
+link:/assets/examples/posts/quarkus-native-neo4j-call-tree/import.cypher[this link]
```
[the remark this answers is not in the document]
Comment (Karm) [reply _posts/2021-11-30-quarkus-native-neo4j-call-tree.adoc:89]: Yes. It fixes a dead link.

File: _posts/2023-06-23-podman-for-windows.adoc (hunk 1 above)
Comment (Karm) [reply _posts/2023-06-23-podman-for-windows.adoc]: Ack.

**Comment by Karm** [comment]
@cescoffier , @gastaldi  done :+1:

**Pull request merged**